src/toolbar/toolbar-controller.js

```diff
@@ -59,7 +59,6 @@
 
         normalSizeClick() {
             this.window.restore();
-            this.window.resizeTo(defaultWidth, defaultHeight, 'top-right');
```

Comment:
Actually the issue I was talking about no longer seems to be present. As I recall the steps to break it were:

- click maximise icon

- click to compact icon

- click expand icon

- click restore icon. This would end up back in a psuedo compact view.

but now clicking the expand icon in a compact window that was previously maximised does not return it to the maximised state so this isn't an issue. Although the fact that it doesn't return to a maximised state may be an issue unto it's own.